config/crd/kustomization.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0-rc1
-- https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v0.5.0-rc1/config/crd/experimental/gateway.networking.k8s.io_referencepolicies.yaml
```

Comment:
This had been necessary to install the ReferencePolicy CRD by default, as it had seemed to be excluded when only installing the experimental channel defaults due to being marked as deprecated - is that no longer the case?

Reply:
@mikemorris https://github.com/kubernetes-sigs/gateway-api/pull/1216 restored the `ReferencePolicy` CRD to the experimental channel